.eslintrc.js

```diff
@@ -19,9 +20,6 @@ module.exports = {
     // JSHint "proto", disabled due to warnings
     'no-proto': 0,
 
-    // JSHint "strict"
-    'strict': [2, 'global'],
```

Comment:
So if I add that, then I get:

```
> ember-cli-code-coverage@0.3.12 lint /Users/paulw/Work/stuff/ember-cli-code-coverage
> eslint config lib test tests *.js


/Users/paulw/.../ember-cli-code-coverage/config/environment.js
  2:1  error  'use strict' is unnecessary inside of modules  strict

/Users/paulw/.../ember-cli-code-coverage/config/release.js
  1:1  error  'use strict' is unnecessary inside of modules  strict

/Users/paulw/.../ember-cli-code-coverage/lib/attach-middleware.js
  1:1  error  'use strict' is unnecessary inside of modules  strict

....
```

So which do you want, do you want the eslint rule or do you want the "use stricts"? Personally I'd rather have it hue closer to the default ember-cli blueprint, but that's just my opinion.